projects/canopy/src/lib/card/docs/card/card.stories.ts

```diff
@@ -437,3 +438,133 @@ dataPointsCard.args = {
     },
   ],
 };
+
+const cardGroupTemplate = `
+<div lgContainer>
+  <div lgRow>
+    <div [lgCol]="12">
```

Comment:
Total nitpick and really doesn't matter, but this could be `lgCol="12"`. Same in the block further down, too

Reply:
Updated 👍